src/test/java/org/apache/sysds/test/functions/mlcontext/MLContextMLogregTest.java

```diff
@@ -60,30 +60,30 @@ public void testMLogregDense() {
 
 	private void runMLogregTestMLC(boolean sparse) {
 
-		baseDirectory = "target/testTemp/functions/mlcontext/";
 
 		//generate actual datasets
 		double[][] X = getRandomMatrix(rows, cols, 0, 1, sparse?sparsity2:sparsity1, 2384);
-		writeInputMatrixWithMTD("X", X, true);
 		double[][] y = TestUtils.round(getRandomMatrix(rows, 1, 0.51, 5+0.49, 1.0, 9283));
-		writeInputMatrixWithMTD("Y", y, true);
+
+		baseDirectory = "target/testTemp/functions/mlcontext/";
```

Comment:
- Based on the `baseDirectory`, the `fullDMLScriptName` will be influenced.
- and input/output data will be available at `baseDirectory/in/`, `baseDirectory/out/` respectively.